app/packages/core/src/components/ColorModal/ColorModal.tsx

```diff
@@ -22,6 +22,7 @@ import {
 import SidebarList from "./SidebarList";
 import { activeColorEntry } from "./state";
 import { ACTIVE_FIELD } from "./utils";
+import styles from "../../../../../packages/components/src/scrollable.module.css";
```

Comment:
Is it possible to use `import { scrollbarStyles } from "@fiftyone/components"`? The styles may need to be added as an export in  `@fiftyone/components/index.ts`

Reply:
Yes, it's already exported.